docs/user/openstack/external_load_balancer.md

```diff
@@ -0,0 +1,224 @@
+# Deploying OpenShift with an external load balancer
+
+This document explains how to deploy OpenShift with an external load balancer, rather than with the internal (that is based on HAproxy & Keepalived).
+
+## Table of Contents
+- [Deploying OpenShift with an external load balancer](#deploying-openshift-with-an-external-load-balancer)
+  - [Table of Contents](#table-of-contents)
+  - [Common prerequisites](#common-prerequisites)
+  - [Deploy the load-balancer](#deploy-the-load-balancer)
+  - [Deploy OpenShift](#deploy-openshift)
+  - [Known limitations](#known-limitations)
+  - [Notes](#notes)
+
+## Common prerequisites
+
+* When deploying OpenShift with an external load balancer, it's required to bring your own network(s) before
+  the deployment. This can be a tenant network or a provider network.
+* The load balancer(s) will have to be deployed before installing OpenShift.
+  * It has be connected to the network(s) where OpenShift will be deployed.
+  * If it's a server managed by OpenStack, allowed address pairs have to be configured
+    for the port that will serve API and Ingress traffic, otherwise the VIP traffic will be rejected by the
+    OpenStack SDN.
+  * The firewall has to allow the following traffic on the load balancer (if a server managed by OpenStack, create a security group):
+    * 22/TCP - SSH
```

Comment:
I understand it may be necessary to open SSH traffic from certain IP range to configure the LB via the ansible script, but is it needed afterward, or from any address? If it is not strictly necessary, perhaps we should say so.